<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>607</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>D2Coding</family>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>571</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>도서 상세 정보</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>341</width>
            <height>221</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>제목:    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_name">
               <property name="placeholderText">
                <string>제목 입력</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>저자:    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_author">
               <property name="readOnly">
                <bool>false</bool>
               </property>
               <property name="placeholderText">
                <string>저자 입력</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>출판사:  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_pub">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>장르:    </string>
               </property>
               <property name="textFormat">
                <enum>Qt::AutoText</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_div">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>상태:    </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_con">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>대출정보:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="input_std_loan">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>360</x>
            <y>20</y>
            <width>160</width>
            <height>221</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_clr">
             <property name="text">
              <string>입력초기화</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_sea">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="text">
              <string>조회</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_bor">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>대출</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_ret">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>반납</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QTableWidget" name="tblbook">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>9999999</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
